deploy/runtime/ui/Solar Resource Data.txt

```diff
@@ -2477,7 +2477,7 @@ TextEntry
 solar_resource
 3
 4
-Location
+Solar resource library current selection
```

Comment:
Do you want to change in this pull request?

Reply:
Yes. That addresses Item 1 in #564.